### PR TITLE
Include elastic_integration plugin.

### PR DESCRIPTION
**Pull request by mashhurs**
### Description
Include `elastic_integration` plugin manually for now.

### CODE REVIEW

File: docs/plugins/filters/elastic_integration.asciidoc [hunk 1]
```diff
@@ -0,0 +1,528 @@
+:plugin: elastic_integration
+:type: filter
+:default_plugin: 1
```
Comment (karenzone) [inline docs/plugins/filters/elastic_integration.asciidoc:3]: @mashhurs, I know that you generated this locally for `main`.  This plugin will probably be bundled by default in `main`, but isn't yet.  Seems like maybe we should set the `default_plugin` value to `0` until then?  Please LMKWYT.

Comment (mashhurs) [reply docs/plugins/filters/elastic_integration.asciidoc:3]: Ah thanks for catching this up, it is due to my local and let me update it.

Comment (mashhurs) [inline docs/plugins/filters/elastic_integration.asciidoc:3]:
```suggestion
:default_plugin: 0
```

Comment (karenzone) [reply docs/plugins/filters/elastic_integration.asciidoc]: Thanks for confirming. Nice to be sure that we're on the same page.

**Review by karenzone — APPROVED** [review]
LGTM. Thanks!

**Comment by karenzone** [comment]
50423

**Pull request merged**